core/src/main/java/com/klaytn/caver/crypto/KlayCredentials.java

```diff
@@ -63,6 +103,52 @@ public static KlayCredentials create(ECKeyPair ecKeyPair, String address) {
         return new KlayCredentials(ecKeyPair, address);
     }
 
+    public static KlayCredentials create(List<ECKeyPair> ecKeyPair, String address) {
```

Comment:
It seems Builder pattern can be adjust to this class. This pattern is really effective when there are many combination of creating certain object.

Reply:
Applying build patten is a good idea. However, this implementation will go as closely as possible to existing usage. If rearchitecturing is done, then consider.